### DISCUSSION TITLE
[BP#37] Préférer les CSS aux images? Ca dépend...

### DISCUSSION THREAD

**Discussion by ldevernay** [cnumr/best-practices#546] [Bonnes pratiques]
Il faudrait préciser l'application de cette bonne pratique. Sur des images complexes, je ne suis pas sûr que ce soit avantageux. Plus généralement, il faudrait étudier l'impact CPU/mémoire d'utiliser du CSS plutôt qu'une image.

**Comment by rlemaire** [comment]
L'idée est d'utiliser du CSS pour les ombres, dégradés ou bordures arrondies. La pratique date d'une éopoque où la possilité de le faire en CSS était nouvelle.
Il n'est pas question de remplacer de vraies images par du code css.

**Reply by ldevernay** [reply]
OK, j'avais eu le doute à la lecture mais ça venait probablement de ma propre interprétation. 
Je serais curieux de vérifier par la mesure si les effets CSS sont véritablement plus avantageux qu'une nouvelle image bien optimisée mais c'est un autre sujet (et du pinaillage).
